git_machete/cli.py

```diff
@@ -231,6 +231,7 @@ def create_cli_parser() -> argparse.ArgumentParser:
     github_parser.add_argument('--by')
     github_parser.add_argument('--draft', action='store_true')
     github_parser.add_argument('--mine', action='store_true')
+    github_parser.add_argument('-y', '--yes', action='store_true')
```

Comment:
Is this option documented? 🤔 It's pretty unclear from just the looks of it what `git machete github sync --yes` would do as opposed to `git machete github sync`

Reply:
Also, is there a logic to autocomplete that option/make sure it's only passed to `sync` and not to other commands?

Reply:
In fact, it's probably okay NOT to support this branch for `github sync` to avoid bloating the CLI